scripts/stat.sh

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env bash
+
+# stat.sh - Check foundational node status/statistics (requires 'bc' installed)
+
+# Get timestamps
+lastbingo=`grep -i bingo latest/*.log |tail -n 1 |cut -f 2 -d 'r' |cut -c 16-34 | tr T \ `
+time1=$(date -d "$lastbingo" +%s)
+rawtime=`date +%s`
+time2=$(($rawtime - 60))
+created=`grep -i -m 1 allocated latest/*.log |cut -f 7-9 -d ":" |cut -f 1 -d "." |cut -c 2- | tr T \ `
+latest=`grep -i bingo latest/*.log |tail -n 1 |cut -f 2 -d 'T' |cut -c 1-14`
+bingos=`grep -i bingo latest/*.log |tail -n 2 |cut -f 2 -d 'T' |cut -c 7-14`

```

Comment:
duplicated grep query, why don't you just use a temporary file to store all the 'grep -i bingo' ?